### PR TITLE
Feat: add feature delete object store

### PR DESCRIPTION
**Pull request by aparzi**
Hi @assuncaocharles,
this pull request adds the ability to delete a store.

### CODE REVIEW

**Comment by aparzi** [comment]
@assuncaocharles this PR resolve issue #274.

**Comment by aparzi** [comment]
Hi @assuncaocharles, 
any update for this PR?

File: projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts [hunk 1]
```diff
@@ -570,4 +570,12 @@ export class NgxIndexedDBService {
         .catch((reason) => obs.error(reason));
     });
   }
+
+  /**
+   * Delete the store by name.
+   * @param storeName The name of the store to query
+   */
+  deleteObjectStore(storeName: string): void {
+    DeleteObjectStore(this.dbConfig.name, ++this.dbConfig.version, storeName);
```
Comment (assuncaocharles) [inline projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts]: This breaks a little bit the pattern we are establishing, can we return a observable here ?

**Comment by assuncaocharles** [comment]
Hey @aparzi just one minor thing and we will be good to merge.

File: projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts (hunk 1 above)
Comment (aparzi) [reply projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts]: New commit done use Observable pattern.

**Comment by aparzi** [comment]
Hi @assuncaocharles,
use observable pattern.

**Comment by aparzi** [comment]
@assuncaocharles any news?

**Review by assuncaocharles — APPROVED** [review]

**Pull request merged**